package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-newrelic-lambda-layers",
-  "version": "0.4.3",
+  "version": "0.4.4",
```

Comment:
Updating the dependency major version like this is beyond the scope of a bugfix change. It may introduce incompatibilities for existing users. We should update the major version number.

Reply:
@MattWhelan 1.0.0?